spec/lib/submission_builder/ty2024/states/nj/documents/nj1040_spec.rb

```diff
@@ -608,13 +608,50 @@
       end
     end
 
+    describe 'line 45 - balance of tax' do
+      let(:intake) { create(:state_file_nj_intake) }
+      it 'sets line 45 to calculated value' do
+        stub_value = 20_000
+        allow_any_instance_of(Efile::Nj::Nj1040Calculator).to receive(:calculate_line_45).and_return stub_value
+        expect(xml.at("BalanceOfTaxA").text).to eq(stub_value.to_s)
+      end
+    end
+
+    describe 'line 49 - total credits' do
+      let(:intake) { create(:state_file_nj_intake) }
+      it 'sets line 49 to calculated value' do
+        stub_value = 0
```

Comment:
@embarnard There's a potential future where it might not always return 0 if we expand the scope. I'd rather stub it now rather than having this test fail if we ever do change that functionality.